2_generics_collections/generic_binary_search/src/lib.rs

```diff
@@ -0,0 +1,101 @@
+/// Finds the index of `value` in the sorted slice `v`
+/// Note: `value` will always implement the `Copy` Trait
+/// Returns the index if found, None otherwise
+pub fn generic_binary_search<?>(v: &[?], value: ?) -> Option<usize> {
+    unimplemented!();
+}
```

Comment:
You should put that function in comments, just like for the `ordered_generic_vec` exercise. This way, `rustfmt` will not complain, as it tries to parse the file first and fails if the syntax is invalid